lib/tasks/rake_helpers/user_management.rb

```diff
@@ -34,4 +45,14 @@ def self.create_csv_for(users, filename)
       end
     end
   end
+
+  def self.disable(users)
+    error_count = 0
+    users.each do |user|
+      user.update!(disabled_at: DateTime.now)
```

Comment:
I missed this with the last review. Can we use this instead;

```suggestion
      user.disable
```

Also, in the queries above you could use `User.disabled.where(...)` instead of `User.where(...).where.not(disabled_at: nil)`.

(See `app/models/concerns/disablable.rb`)

Reply:
Good spot. Thanks.